### ISSUE TITLE
Waiting Room Design Update

### ISSUE THREAD

**Issue by sebasnieb** [blopker/totem#116]
So, as we have changed signifcantly the UI while integrating the Video functionality.
Based on those changes the "Wainting Room" the UI has to change in order to reflect the same logic as it is in the Video version of the app.
So, for the "Waiting room " as Guest perspective : 
https://www.figma.com/file/Q8ZcLftDd8JQ9HIFz0DZJD/Totem-app?node-id=0%3A429
Long title :
https://www.figma.com/file/Q8ZcLftDd8JQ9HIFz0DZJD/Totem-app?node-id=715%3A11864

Keeper
https://www.figma.com/file/Q8ZcLftDd8JQ9HIFz0DZJD/Totem-app?node-id=576%3A12578

FYI @blopker

**Comment by blopker** [comment]
Taking off eng ready since this is a larger change and it needs more details about what needs to be done.

**Comment by sebasnieb** [comment]
Would you need me to explain the above changes and comparison?

**Comment by blopker** [comment]
Yes, the devs can eyeball it, but they will likely not see everything you intend to change. To get close to what you want, a list of expected changes here will help. Also, sometimes annotating the mocks can be even better.
